src/pages/[platform]/build-a-backend/existing-resources/cli/index.mdx

```diff
@@ -0,0 +1,996 @@
+
+## Connect mobile app to existing AWS resources
+
+This guide will walk you through connecting a new Flutter app to AWS resources created with Amplify for an existing Flutter app. If you don't already have an existing app, you can follow this [Flutter tutorial](https://docs.amplify.aws/start/getting-started/setup/q/integration/flutter/) to create a budget tracker app that uses Amplify Auth and API resources.
+
+Before you begin, you will need:
+
+* An existing Flutter app
+* An AWS account: If you don't already have an account, follow the [Setting Up Your AWS Environment](https://aws.amazon.com/getting-started/guides/setup-environment/) tutorial for a quick overview.
+* [Install](https://docs.amplify.aws/cli/start/install/) and configure the Amplify CLI.
+* [Install](https://docs.flutter.dev/get-started/install) and configure Flutter.
+* A text editor combined with Flutter’s command-line tools. For this guide, we will use VS Code, but you can use your preferred IDE.
+
+### Find the AWS backend details
+Before connecting the backend resources to our new app, we first need to locate the details of the AWS environment provisioned for the existing app.
+
+**Step 1:** In your existing app, open the file `<Your-App>/amplify/team-provider-info.json`.
+
+<img src="/images/existing-resources/team-provider-info-mobile-cli.png" alt="The team-provider-info.json file within the file directory of the Amplify app." width="450" />
+
+**Step 2:** In the `team-provider-info.json` file, note the following:
+
+1. The environment you want to use 
+2. The `AmplifyAppId` for the required environment
+
+{/* cSpell:disable */}
+<img src="/images/existing-resources/dev-amplifyappid-mobile-cli.png" alt="The environment and AmplifyAppId in team-provider-info.json file." width="450" />
+{/* cSpell:enable */}
+
+### Create the Flutter app
+Now that we have gathered the necessary backend details, we can start building out the new Flutter app.
+
+**Step 1:** Create a Flutter app by running the following command in your terminal.
+
+```bash showLineNumbers={false}
+
+flutter create amplify_connect_resources 
+
+```
+
+
+
+**Step 2:** Open the newly created Flutter app using VS Code by running the following commands in your terminal. 
+
+```
+cd amplify_connect_resources 
+code . -r 
+```
+
+![Open the created app using VS Code.](/images/existing-resources/app-vscode-mobile-cli.png)
```

Comment:
Is this image necessary or is there something more we want to point out for the user to review here? Should I be looking at the menu details? If so then we may want an image that has better readability of that menu.